test-runner/gh-actions-runner.service

```diff
@@ -0,0 +1,26 @@
+# Install with:
+#   sudo install -m 644 gh-actions-runner.service /etc/systemd/system/
+#   sudo systemctl daemon-reload
+#   sudo systemctl enable gh-actions-runner
+# Run with:
+#   sudo systemctl start gh-actions-runner
+# Stop with:
+#   sudo systemctl stop gh-actions-runner
+# See live logs with:
+#   journalctl -f -u gh-actions-runner.service --no-hostname --no-tail
+[Unit]
+Description=GitHub Actions Runner Container
+After=docker.service
+Requires=docker.service
+[Service]
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker compose stop %N
+ExecStartPre=-/usr/bin/docker compose rm %N
+ExecStartPre=-/usr/bin/docker pull lurklab/gh-actions-runner:latest
+ExecStart=/usr/bin/docker compose \
```

Comment:
I would put the path of `/usr/bin/docker` in a variable, e.g. with `Environment="DOCKER=/usr/bin/docker`, and then using ${DOCKER}.

Reply:
I was unable to get the `${DOCKER}` variable recognized when it was the first argument in `ExecStartPre=`. From the "Command Lines" section of the `systemd.service(5)` man page:

> Note that the first argument (i.e. the program to execute) may not be a variable.

Are there any workarounds you are aware of?

Reply:
There isn't, sorry, it sounds like this was a bad idea on my part, thanks for trying.